src/Parser.jl

```diff
@@ -322,9 +322,9 @@ the byte before `to`. Bytes enclosed should all be ASCII characters.
 """
 function int_from_bytes(bytes::Vector{UInt8}, from::Int, to::Int)
     @inbounds isnegative = bytes[from] == MINUS_SIGN ? (from += 1; true) : false
-    num = Int64(0)
+    num = Int(0)
```

Comment:
Yes, you are right. Same is true two lines down, I guess, so I removed the ``Int`` there as well.